css/styles.css

```diff
@@ -0,0 +1,146 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: lightgray; 
+  font-family: sans-serif;
+  font-size: 19px;
+  margin-top: 0px;
+}
+
+.container {
+  width: 800px;
+  margin: 0 auto;
+}
+
+header {
+  background-color: black;
+  border-bottom: 4px solid #f8ff00;
+  padding: 0px 25px;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+header a {
+  color: white;
+  margin-right: 12px;
+}
+
+nav {
+  text-align: right;
+  margin-bottom: 15px;
+}
+
+h1 {
+  color: #f8ff00; 
+  padding: 15px 0px 0px 15px;
+  font-family: "ogham";
+  margin-top: 0px;
+}
+
+h2 {
+  margin-bottom: 5px;
+}
+
+.caps {
+  text-transform: uppercase;
+  font-weight: bold;
+  font-size: 20px;
+}
+
+.slogan {
+  background-color: #c6c6c6;
+  text-align: center;
+  font-style: italic;
+  padding: 5px;
+  font-size: 27px;
+  font-weight: bold;
+  font-family: serif;
+}
+
+.crossout {
+  color: darkgray;
+  text-decoration: line-through;
+}
+
+.underline {
+  text-decoration: underline;
+  text-decoration-color: #f8ff00;
+}
+
+article {
+  background-image: url("../images/coworking.jpg");
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: 100%;
```

Comment:
@andrewhinderaker this could have been written with the background shorthand 
`background: url("../images/coworking.jpg") no-repeat center center;
  background-size: 100%;`